projects/lights_control/inc/lights_blinker.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
```

Comment:
Please add a module description and capitalize your comments.

Reply:
done 🤗